src/components/Button/_Button.scss

```diff
@@ -1,48 +1,12 @@
+// Keeping `button-base` and `button--outline` for the
+// Accordion component until that component is updated.
 @mixin button-base {
-  @include space-inset-xs;
-
-  border-radius: 2px;
+  border-radius: 2px !important;
   cursor: pointer;
   display: flex;
   justify-content: center;
-  padding: var(--space-xs) var(--space-s);
+  padding: var(--space-xs) var(--space-s) !important;
```

Comment:
What is necessitating the `!important`?

Reply:
This is only really being used in the `Accordion` component to style the input. Some of Chakra's inline css rules take precedence so it needs the `!important`. But once we update the `Accordion` component, this can be removed.